source/reference/minio-mc/mc-event-remove.rst

```diff
@@ -66,55 +65,91 @@ Parameters
 ~~~~~~~~~~
 
 .. mc-cmd:: ALIAS
+   :required:
 
-   *Required* The S3 service :ref:`alias <alias>` and bucket from
-   which the command removes the event notification. For example:
+   The S3 service :ref:`alias <alias>` and bucket from which the command removes the event notification. 
+   For example:
 
    .. code-block:: shell
 
       mc event add play/mybucket
 
 .. mc-cmd:: ARN
+   :required:
+
+   The :aws-docs:`Amazon Resource Name (ARN) <IAM/latest/UserGuide/reference-arns>` of the notification target.
 
-   *Required* The :aws-docs:`Amazon Resource Name (ARN)
-   <IAM/latest/UserGuide/reference-arns>` of the notification target.
+   The MinIO server outputs an ARN for each configured notification target at server startup. 
+   See :ref:`minio-bucket-notifications` for more information.
 
-   The MinIO server outputs an ARN for each configured
-   notification target at server startup. See
-   :ref:`minio-bucket-notifications` for more
-   information.
+   Retrieve the ARN by running :mc:`mc event ls` on the bucket.
 
 .. mc-cmd:: --event
+   :optional:
+
+   The event type(s) specified when adding the event. 
```

Comment:
could be more explicit that this is about something that happened in the past. maybe?
```suggestion
   The event type(s) originally specified when the event was added. 
```

Reply:
Also: you can check the event's types first right?

Reply:
I try to avoid passive past tense, but I think it makes sense here.